speeduino/secondaryTables.ino

```diff
@@ -12,7 +12,7 @@ void calculateSecondaryFuel()
     {
       currentStatus.VE2 = getVE2();
       //Fuel 2 table is treated as a % value. Table 1 and 2 are multiplied together and divded by 100
-      uint16_t combinedVE = ((uint16_t)currentStatus.VE1 * (uint16_t)currentStatus.VE2) / 100;
+      uint16_t combinedVE = (uint16_t)currentStatus.VE1 + ((uint16_t)currentStatus.VE1 * ((uint16_t)currentStatus.VE2 / 100));
```

Comment:
This is not correct, there are at least 2 things going on here. 

1) The MULTIPLY mode for these tables works by using the secondary table as a percentage. 
Eg: If you have 80 in the primary table and 50 in the secondary, the resulting value should be 40 (50% of 80)

2) `((uint16_t)currentStatus.VE2 / 100))` will not work. This is all integer math, not floats, so any value below 100 will return 0, between 100 and 200 will return 1 etc.